tutorials/01_GettingREMIND.md

```diff
@@ -15,6 +15,7 @@ REQUIREMENTS
 - Windows only: RTools
 - LaTeX
 - pandoc
+- for couplings to other models: Python >= 3.7
```

Comment:
only relevant for coupling with models which need Python, maybe specify here?

Reply:
I think this describes current situation better

Reply:
If it helps to have an "initial egg", one of the first Python things we will likely do is integrate the [climate-assessment](https://github.com/iiasa/climate-assessment) toolchain. It requires just Python 3, works with 3.7 and recommends 3.9 (which is what they test on). So I think requiring >=3.7 is quite reasonable for now.